Fix:

error: implicitly declaring library function 'strcmp' with type 'int (const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'strncmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'gd_parse' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- ext/getdate/getdate.y.orig	2006-10-24 00:15:52.000000000 -0500
+++ ext/getdate/getdate.y	2023-08-29 04:39:29.000000000 -0500
@@ -8,6 +8,10 @@
 **  This code is in the public domain and has no copyright.
 */
 
+#define HAVE_MEMCPY 1
+#define HAVE_STDLIB_H 1
+#define STDC_HEADERS 1
+
 #ifdef HAVE_CONFIG_H
 # include <config.h>
 # ifdef HAVE_ALLOCA_H
